src/patternfly/components/TreeView/examples/TreeView.md

```diff
@@ -716,9 +818,15 @@ beta: true
 | `.pf-c-tree-view__node-toggle` | `<div>` | Initiates a tree view toggle. |
 | `.pf-c-tree-view__node-toggle-button` | `<button>` | Initiates a tree view toggle button. |
 | `.pf-c-tree-view__node-toggle-icon` | `<span>` | Initiates a tree view toggle icon. |
+| `.pf-c-tree-view__node-title` | `<span>` | Initiates a tree view node title. |
 | `.pf-c-tree-view__node-text` | `<span>` | Initiates tree view text. |
 | `.pf-c-tree-view__node-icon` | `<span>` | Initiates a tree view icon. |
 | `.pf-c-tree-view__node-check` | `<span>` | Initiates a tree view check. |
 | `.pf-c-tree-view__action` | `<div>` | Initiates a tree view action wrapper. |
 | `.pf-c-tree-view__search` | `<div>` | Initiates a tree view search wrapper. |
+| `.pf-m-bordered` | `.pf-c-tree-view` | Modifies the tree view to the bordered presentation. |
+| `.pf-m-compact` | `.pf-c-tree-view` | Modifies the tree view to the compact presentation. |
+| `.pf-m-background-transparent` | `.pf-c-tree-view.pf-m-compact` | Modifies the tree view compact variant node containers to have a transparent background. |
 | `.pf-m-current` | `.pf-c-tree-view__node` | Modifies the tree view node to be current. |
+| `.pf-m-truncate-text` | `.pf-c-tree-view` | Modifies the tree view text to truncate. |
```

Comment:
Do this and `.pf-m-truncate` do the same thing, except this applies it to all text elements in the tree vieww, and `.pf-m-truncate` applies to an individual element? If so I wonder if we could use the same modifier name for both, similar to how we do on table?